glotaran/testing/plugin_system.py

```diff
@@ -49,10 +49,10 @@ def _monkeypatch_plugin_registry(
     monkeypatch_plugin_registry_project_io
     """
     if test_registry is not None:
-        initila_plugins = {} if create_new_registry else __PluginRegistry.__dict__[register_name]
+        initial_plugins = {} if create_new_registry else __PluginRegistry.__dict__[register_name]
```

Comment:
whoops, funny typo, good catch ^^

Reply:
Wouldn't have seen it w/o spell checker 😅